doc/TOPP_tutorial/TOPP_scripting.doxygen

```diff
@@ -34,6 +34,269 @@
+                  million and 'fragment_mass_error_da'for the FragmentMassError in Dalton into the first hit of the peptideIdentifications.<br>
+                  <b> Other Output:</b> <br>
+                  The FragmentMassError metric stores the average and the variance of FragmentMassErrors in ppm.
+             </TD>
+        </TR>
+        <TR>
+            <TH colspan="2">
+                <br><b>MissedCleavages</b>
+            </TH>
+        </TR>
+        <TR>
+              <TD>
+                   <b>Essential input data:</b> PostFDR FeatureXML
+              </TD>
+              <TD>
+                   <b>Output</b>
+              </TD>
+        </TR>
+        <TR>
+              <TD>
+                   @b Brief: <br>
+                   This metric counts the number of MissedCleavages per PeptideIdentification in a FeatureMap.
+                   and returns an agglomeration statistic (observed counts).<br>
+                   @b Description: <br>
+                   This MissedCleavages metric counts the number of MissedCleavages per PeptideIdentification given a FeatureMap
+                   and returns an agglomeration statistic (observed counts).
+                   Additionally the PeptideHits in the FeatureMap are augmented with MetaInformation.
+              </TD>
+
+              <TD>
+                   <b> Changes in files:</b> <br>
+                   This metric adds the new metavalues 'missed_cleavages' to the first peptidehit in each peptideidentification
+                   of the featureMap. <br>
+                   <b> Other Output:</b> <br>
+                   The MissedCleavages metric stores the number of cuts of all peptides in a map.
+              </TD>
+        </TR>
+        <TR>
+            <TH colspan="2">
+                <br><b>MS2IdentificationRate</b>
+            </TH>
+        </TR>
+        <TR>
+              <TD>
+                   <b>Essential input data:</b> PostFDR FeatureXML, raw mzML file
+              </TD>
+              <TD>
+                   <b>Output</b>
+              </TD>
+        </TR>
+        <TR>
+              <TD>
+                   @b Brief: <br>
+                   This metric computes the MS2IdentificationRate.<br>
+                   @b Description: <br>
+                   The MS2IdentificationRate metric calculates the Rate of the MS2Identification as follows:
+                   The number of all peptideIdentification are counted and that number is devided by the number of
+                   counted ms2 spectra.
+              </TD>
+
+              <TD>
+                   <b> Changes in files:</b> <br>
+                   This metric does not change anything in the data.<br>
+                   <b> Other Output:</b> <br>
+                   The MS2IdentificationRate metric stores the number of peptideIdentifications, the number of ms2 spectra
```

Comment:
`The MS2IdentificationRate metric returns the number of peptide identifications, ...`